applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/snats.js

```diff
@@ -166,6 +169,20 @@ return view.extend({
 		o.default = o.enabled;
 		o.editable = true;
 
+		o = s.taboption('general', form.ListValue, 'family', _('Restrict to address family'));
+		o.modalonly = true;
+		o.rmempty = true;
+		o.value('', _('IPv4 only')); //assume it is ipv4 if empty
+		o.value('ipv4', _('IPv4 only'));
+		o.value('ipv6', _('IPv6 only'));
+		o.validate = function(section_id, value) {
+			if (value != 'ipv4' && value != 'ipv6')
+				value = 'ipv4'
```

Comment:
This can be removed, if `value` is the empty string, we should add host hints for both IPv4 and IPv6.